packages/features/data-table/components/DataTable.tsx

```diff
@@ -152,7 +164,7 @@ export function DataTable<TData, TValue>({
                             width: `calc(var(--col-${cell.column.id}-size) * 1px)`,
                           }}
                           className={classNames(
-                            "flex shrink-0 items-center overflow-auto",
+                            "flex shrink-0 items-center overflow-hidden",
```

Comment:
Having scroll in individual cells are visually bad on initial rendering (the scroll bars in all the columns appear briefly on load and disappear soon).

We can hide the overflown part of the cell for now, which can be a downside, but we should introduce column resizing feature soon to solve this problem.

(Another solution to this is to make column sizes change according to the content, but that was one of the root cause of flaky scroll)